Library/Homebrew/os/linux/elf.rb

```diff
@@ -68,6 +68,12 @@ def arch
     end
   end
 
+  def arch_compatible?(wanted_arch)
+    return true unless elf?
+
+    wanted_arch == arch
```

Comment:
Technically isn't correct for PPC but we can follow up with that.

Reply:
Going to punt on this!